aspnetcore/security/key-vault-configuration.md

```diff
@@ -15,73 +15,100 @@ uid: security/key-vault-configuration
+# [ASP.NET Core 1.1](#tab/aspnetcore11)
 
-This document explains how to use the [Microsoft Azure Key Vault](https://azure.microsoft.com/services/key-vault/) configuration provider to load application configuration values from Azure Key Vault. Azure Key Vault is a cloud-based service that helps you safeguard cryptographic keys and secrets used by apps and services. Common scenarios include controlling access to sensitive configuration data and meeting the requirement for FIPS 140-2 Level 2 validated Hardware Security Modules (HSM's) when storing configuration data.
+View or download sample code for 1.1:
+
+* [Basic sample](https://github.com/aspnet/Docs/tree/master/aspnetcore/security/key-vault-configuration/samples/sample1/1.1) - Reads secret values into an app.
+* [Key name prefix sample](https://github.com/aspnet/Docs/tree/master/aspnetcore/security/key-vault-configuration/samples/sample2/1.1) - Reads secret values using a key name prefix that represents the version of an app, which allows you to load a different set of secret values for each app version. 
+
+# [ASP.NET Core 2.x](#tab/aspnetcore2x)
+
+View or download sample code for 2.x:
+
+* [Basic sample](https://github.com/aspnet/Docs/tree/master/aspnetcore/security/key-vault-configuration/samples/sample1/2.0) - Reads secret values into an app.
+* [Key name prefix sampl](https://github.com/aspnet/Docs/tree/master/aspnetcore/security/key-vault-configuration/samples/sample2/2.0) - Reads secret values using a key name prefix that represents the version of an app, which allows you to load a different set of secret values for each app version.
+
+---
+
+This document explains how to use the [Microsoft Azure Key Vault](https://azure.microsoft.com/services/key-vault/) configuration provider to load application configuration values from Azure Key Vault secrets. Azure Key Vault is a cloud-based service that helps you safeguard cryptographic keys and secrets used by apps and services. Common scenarios include controlling access to sensitive configuration data and meeting the requirement for FIPS 140-2 Level 2 validated Hardware Security Modules (HSM's) when storing configuration data. This feature is available for applications that target ASP.NET Core 1.1 or higher.
 
 ## Package
-To use the provider, add a reference to the [`Microsoft.Extensions.Configuration.AzureKeyVault`](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureKeyVault/) package. The provider depends on .NET Framework 4.5.1 or .NET Standard 1.5 or higher. This feature is available for applications that target ASP.NET Core 1.1.0 or higher.
+To use the provider, add a reference to the [Microsoft.Extensions.Configuration.AzureKeyVault](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureKeyVault/) package. The provider depends on .NET Framework 4.5.1 (1.1)/4.6.1 (2.x) or .NET Standard 1.5 or higher.
```

Comment:
I recommend including a space on either side of the forward slash. This will make the version numbers a bit easier to read. Also, it may be best to explicitly state "ASP.NET Core 1.1" and "ASP.NET Core 2.x" instead of 1.1 and 2.x, respectively.